static/docs/user-guide/install/plugins.md

```diff
@@ -0,0 +1,30 @@
+# IDE Plugins & Syntax Highlighting
+
+When you add a file or a stage to your pipeline, DVC creates a special
+[DVC-file](/doc/user-guide/dvc-file-format) that contains all the needed
+information to track your data and transformations.
+
+The file itself is in a simple YAML format.
+
+## Vim
+
+In order to recognize DVC-files as YAML in Vim, you should add:
+
+```vim
```

Comment:
Maybe `usage`, `dvc`, or `yaml` (the 3 formats we have custom CSS for) will look better, please try.

Reply:
None of them looks better.